src/it/scala/org/ergoplatform/it/Docker.scala

```diff
@@ -79,8 +79,10 @@ class Docker(suiteConfig: Config = ConfigFactory.empty,
     val containerConfig: ContainerConfig = buildContainerConfig(nodeConfig, ip, restApiPort, networkPort)
     val containerId = Try {
       client.createContainer(containerConfig, configuredNodeName + "-" + hashString)
-    }.recoverWith { case e: ImageNotFoundException =>
-      Failure(new Exception(s"Error: docker image is missing ($e)\nRun 'sbt it:test' to generate it."))
+    }.recoverWith {
+      case e: ImageNotFoundException =>
+        Failure(new Exception(s"Error: docker image is missing ($e)\nRun 'sbt it:test' to generate it."))
+      case e => Failure(e)
```

Comment:
I don't think that you need it here. It is done by default if partial function doesn't match

Reply:
Indeed. That compiler implementation of pf's `isDefinedAt` did not get under my skin deep enough yet. Thanks. Fixes.